lib/kubeclient/common.rb

```diff
@@ -51,6 +53,9 @@ def initialize_client(
       validate_auth_options(auth_options)
       handle_uri(uri, path)
 
+      @class_owner = class_owner
+      @entities = nil
```

Comment:
@moolitayer why setting this to `nil` when the first thing you do in `discover` is to set it to an empty hash?